### PR TITLE
Reintroduce Aerospike connector

### PR DESCRIPTION
**Pull request by aahmed-se**
The licensing issue with aerospike client has been resolved with a new version release,, we are now introducing the connector again.

### CODE REVIEW

File: pulsar-io/aerospike/pom.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.aerospike</groupId>
+      <artifactId>aerospike-client-bc</artifactId>
+      <version>${aerospike-client.version}</version>
+      <exclusions>
```
Comment (sijie) [inline pulsar-io/aerospike/pom.xml]: I don't think we need this anymore

Comment (aahmed-se) [reply pulsar-io/aerospike/pom.xml]: you mean the exclusions ?

**Comment by aahmed-se** [comment]
I have removed the exlcusions block

**Comment by sijie** [comment]
run cpp tests

**Pull request merged**